examples/api/lib/material/dropdown_menu/dropdown_menu.2.dart

```diff
@@ -0,0 +1,169 @@
+            children: <Widget>[
+              DropdownMenu<String>(
+                requestFocusOnTap: true,
+                initialSelection: list.first,
+                expandedInsets: EdgeInsets.zero,
+                onSelected: (String? value) {
+                  setState(() {
+                    dropdownValue = value!;
+                  });
+                },
+                dropdownMenuEntries:
+                    list.map<DropdownMenuEntry<String>>((String value) {
+                  return DropdownMenuEntry<String>(value: value, label: value);
+                }).toList(),
+              ),
+              const Text('Text cursor is shown when hovering over the DropdownMenu.'),
+            ],
+          ),
+        ),
+        const SizedBox(height: 20),
+        ListTile(
+          tileColor: colorScheme.primaryContainer,
+          title: const Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: <Widget>[
+              Text('enabled: true'),
+              Text('requestFocusOnTap: false'),
+            ],
+          ),
+          subtitle: Column(
+            children: <Widget>[
+              DropdownMenu<String>(
+                requestFocusOnTap: false,
+                initialSelection: list.first,
+                expandedInsets: EdgeInsets.zero,
+                onSelected: (String? value) {
+                  setState(() {
+                    dropdownValue = value!;
+                  });
+                },
+                dropdownMenuEntries:
+                    list.map<DropdownMenuEntry<String>>((String value) {
+                  return DropdownMenuEntry<String>(value: value, label: value);
+                }).toList(),
+              ),
+              const Text('Clickable cursor is shown when hovering over the DropdownMenu.'),
+            ],
+          ),
+        ),
+        const SizedBox(height: 20),
+        ListTile(
+          tileColor: colorScheme.onInverseSurface,
+          title: const Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: <Widget>[
+              Text('enabled: false'),
+              Text('requestFocusOnTap: true'),
+            ],
+          ),
+          subtitle: Column(
+            children: <Widget>[
+              DropdownMenu<String>(
+                enabled: false,
+                requestFocusOnTap: true,
+                initialSelection: list.first,
+                expandedInsets: EdgeInsets.zero,
+                onSelected: (String? value) {
+                  setState(() {
+                    dropdownValue = value!;
+                  });
+                },
+                dropdownMenuEntries:
+                    list.map<DropdownMenuEntry<String>>((String value) {
+                  return DropdownMenuEntry<String>(value: value, label: value);
+                }).toList(),
+              ),
+              const Text('Defeered cursor is shown when hovering over the DropdownMenu.'),
```

Comment:
```suggestion
              const Text('Default cursor is shown when hovering over the DropdownMenu.'),
```
It looks like you meant "deferred" - which might be slightly more accurate than "default", however, I don't think most readers will understand the distinction.